packages/styles/src/components/validation.scss

```
@@ -0,0 +1,68 @@
+@use 'sass:map';
+@use '../tokens/elements';
+@use '../functions/tokens';
+@use './../mixins/icons';
+
+$post-validation-error: #b20000;
+$post-validation-success: #107800;
+$post-device-font-family-default: 'Swiss Post Sans v005';
+$post-device-font-size-9: 14px;
+$post-device-font-weight-regular: 400;
+$post-radio-button-icon-width-inner: 24px;
+$post-radio-button-icon-height-inner: 24px;
+$post-validation-list-gap-inline: 8px;
```

Can you use the `token.get` function to set these variables? https://github.com/swisspost/design-system/blob/main/packages/styles/src/functions/_tokens.scss#L31

It will help implementing the real tokens in the end. 
```suggestion
@use './../functions/tokens';

tokens.$default-map: (
  post-validation-error: #b20000,
  post-validation-success: #107800,
  post-device-font-family-default: 'Swiss Post Sans v005',
  post-device-font-size-9: 14px,
  post-device-font-weight-regular: 400,
  post-radio-button-icon-width-inner: 24px,
  post-radio-button-icon-height-inner: 24px,
  post-validation-list-gap-inline: 8px,
);
```